keras_cv/layers/preprocessing/random_shear.py

```diff
@@ -148,3 +156,185 @@ def get_config(self):
             }
         )
         return config
+
+    def _augment(self, inputs):
+        image = inputs.get("images", None)
+        label = inputs.get("labels", None)
+        bounding_box = inputs.get("bounding_boxes", None)
+        transformation = self.get_random_transformation(
+            image=image, label=label, bounding_box=bounding_box
+        )  # pylint: disable=assignment-from-none
+        image = self.augment_image(image, transformation=transformation)
+        result = {"images": image}
+        if label is not None:
+            label = self.augment_label(label, transformation=transformation)
+            result["labels"] = label
+        if bounding_box is not None:
+            # changed augment_bounding_box requires image as well
+            # for unnormalizing bbox coordinates
+            bounding_box = self.augment_bounding_box(
+                image, bounding_box, transformation=transformation
+            )
+            result["bounding_boxes"] = bounding_box
+        return result
+
+    def augment_bounding_box(self, image, bounding_boxes, transformation):
```

Comment:
Double check with the BaseImageAugmentationLayer’s definition here to make sure argument order and names match.